erdantic/typing.py

```diff
@@ -1,5 +1,5 @@
 from enum import Enum
-from typing import Any, List, Type, Union
+from typing import Any, List, Type, Union, Literal
```

Comment:
According to the [docs](https://docs.python.org/3/library/typing.html#typing.Literal), it looks like `Literal` was introduced to the `typing` module in Python 3.8. I expect then this will error for Python 3.7, and we'll have to instead import it from `typing_extensions`. I think you can probably use a `try`-`except` like some of the other imports in this module.

Reply:
Fixed.